Python/Batch/sample4_job_scheduler.cfg

```diff
@@ -0,0 +1,6 @@
+[DEFAULT]
+shoulddeletejob=false
+poolvmsize=STANDARD_D1_V2
+poolvmcount=1
+shoulddeletecontainer=false
+shoulddeletepool=false
```

Comment:
I think by default this should be true. Thoughts?

Reply:
switched to true